examples/http_jwt_auth/envoy.yaml

```diff
@@ -0,0 +1,89 @@
+static_resources:
+  listeners:
+    - name: authz
+      address:
+        socket_address:
+          address: 0.0.0.0
+          port_value: 18000
+      filter_chains:
+        - filters:
+            - name: envoy.http_connection_manager
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                stat_prefix: ingress_http
+                codec_type: auto
+                route_config:
+                  name: local_route
+                  virtual_hosts:
+                    - name: local_service
+                      domains:
+                        - "*"
+                      routes:
+                        - match:
+                            prefix: "/"
+                          route:
+                            cluster: web_service
+                http_filters:
+                  - name: envoy.filters.http.wasm
+                    typed_config:
+                      "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+                      type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+                      value:
+                        config:
+                          vm_config:
+                            runtime: "envoy.wasm.runtime.v8"
+                            code:
+                              local:
+                                filename: "./examples/http_jwt_auth/main.go.wasm"
+                  - name: envoy.filters.http.router
+
+    - name: staticreply
```

Comment:
do we need to have staticreply in the first place? Having the direct response in the router seems enough to me for this example: https://github.com/tetratelabs/proxy-wasm-go-sdk/blob/main/examples/vm_plugin_configuration/envoy.yaml#L24-L27

Reply:
That's a better idea. OK, I will change to use it.